### ISSUE TITLE
Bria 5 UI issues

### ISSUE THREAD

**Issue by etheric42** [digitalcreations/MaxTo#460]
**Describe the bug**
When using Bria 5 while MaxTo has window transparency, the UI will stop updating and eventually just be replaced with a blank shell.  The program still functions and you can interact with it if you remember where the buttons are supposed to be, but the visual does not update/is blank.

If MaxTo is closed or window transparency is off when Bria is launched, there is no issue.

**To Reproduce**
Launch MaxTo
Ensure Window Transparency when moving windows in MaxTo is on
Launch Bria 5
Make a call with Bria 5
End call with Bria 5
Try do to literally anything with Bria 5.

**Expected behavior**
Bria 5 to either be excepted from window transparency or for it to work with it

**Screenshots**
![Capture](https://user-images.githubusercontent.com/54451322/63612924-1723ab00-c5a5-11e9-8daa-54264a50d329.PNG)

**System information:**
Windows 10 64 bit
MaxTo 2.0.1

**Comment by vegardlarsen** [comment]
@etheric42  What is the executable name for Bria? E.g. the name of the .exe file?

**Comment by gabeng** [comment]
The executable is `Bria5.exe`.

**Comment by vegardlarsen** [comment]
@gabeng I apologize for letting this issue slip; this should have been attempted a long time ago:

Can you try to add the following section to `%LocalAppData%\MaxTo\shims.json`, then restart both MaxTo and Bria 5?

```json
  {
    "id": "bria",
    "descriptions": {
      "en": {
        "title": "Bria 5",
        "subtitle": "Disable window transparency",
        "description": ""
      }
    },
    "iconUrl": null,
    "url": null,
    "canDisable": true,
    "defaultEnabled": true,
    "ignore": false,
    "disableSubclassing": false,
    "disableSnapping": false,
    "disableTransparency": true,
    "emulatesMaximize": false,
    "windowClasses": [],
    "processes": ["bria5"]
  }
```

I cannot test this myself, as Bria is not available in a trial version.

If this works, I can add this as a compatibility shim included with MaxTo by default.

**Comment by etheric42** [comment]
I tested it and it did not work.

**Comment by etheric42** [comment]
Well, let me rephrase that @vegardlarsen ... I edited that json file, rebooted my computer and tried it and it didn't work.  However, whenever I reopen that json file, I find that the Bria clause has disappeared.  I tried this twice.

**Comment by vegardlarsen** [comment]
@etheric42 Try editing it while MaxTo is running, and it should be picked up when you restart Bria.

If the above segment doesn't work, try setting `"disableSubclassing": true` instead, and if that doesn't work, set `"ignore": true`. Please report back the minimum of what worked, so I can implement this by default.

**Comment by vegardlarsen** [comment]
Closing due to lack of feedback.
